shellcheck reported issue in this script: SC2001:style:3:17

```bash
.github/workflows/build_image.yml:84:9: shellcheck reported issue in this script: SC2001:style:3:17: See if you can use ${variable//search/replace} instead [shellcheck]
   |
84 |         run: |
   |         ^~~~

Error: Process completed with exit code 1.
```

--- a/.github/workflows/build_image.yml
+++ b/.github/workflows/build_image.yml
@@ -84,7 +84,7 @@ jobs:
         run: |
           OUTPUT="$(snyk container test ${{ matrix.tag }} --file=${{ matrix.dockerfile }} --severity-threshold=high --fail-on=all || true)"
           # ANSIエスケープコードを削除
-          CLEAN_OUTPUT="$(echo "$OUTPUT" | sed 's/\x1b\[[0-9;]*m//g')"
+          CLEAN_OUTPUT="${OUTPUT//$'\x1b\['[0-9;]*m/}"
           {
             echo "CLI_OUTPUT<<EOF"
             echo "${CLEAN_OUTPUT}"

